scripts/run_lighthouse_tests.py

```
@@ -169,52 +156,134 @@ def get_entity(line: str) -> tuple[str, str] | None:
+        )
+        print('Config: %s\n' % config_filename)
+        _run_lighthouse_checks_for_config(config_filename)
+        print('\033[1;32m%s checks passed.\033[0m\n' % form_factor)
+
+    pages_count = len(os.environ['LIGHTHOUSE_URLS_TO_RUN'].split(','))
+    all_pages_count = len(os.environ['ALL_LIGHTHOUSE_URLS'].split(','))
+    separator = '=' * 70
+    print(
+        '\n\033[1;36m%s\n'
+        '  LIGHTHOUSE SUMMARY\n'
+        '%s\033[0m' % (separator, separator)
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)